packages/react-sdk/README.md

```diff
@@ -209,6 +209,24 @@ import { OptimizelyExperiment, OptimizelyVariation } from '@optimizely/react-sdk
 </OptimizelyFeature>
 ```
 
+### Rollout a feature user-by-user
+
+To rollout a feature by user rather than by random percentage, setup an attribute in Optimizely and create an Audience, which uses that attribute. Then pass along this attribute to the Provider.
+```
+  <OptimizelyProvider
+    optimizely={optimizely}
+    timeout={500}
+    userId={'user123'}     // UserId used for random percentage rollout 
+    userAttributes={{ 
+      user_id: 'user123'   // Attribute used for non-random audience rollout
```

Comment:
seems a little weird that this is totally tied to a rollout.  Could it say rollout or experiment.

As is is fine with me though.

Reply:
Updated to include `experiment`